pysd/py_backend/utils.py

```diff
@@ -291,6 +304,43 @@ def make_python_identifier(string, namespace=None, reserved_words=None,
     return s, namespace
 
 
+def make_add_identifier(identifier, build_names):
```

Comment:
I'm not sure I understand what this is for or the use case- is there something I should read in the docs, or other places in the code?

Reply:
The external data is merged in one object using the .add method. You can check the final result in test_get_lookups_data_3d_xls.py translated file from test-models repo.
In this case the name of the object should be the same for all of them. Then, to identify external data from the same object as the identifier should be unic we use
```
_ext_lookup_matrix
_ext_lookup_matrixADD1
_ext_lookup_matrixADD2
....
```
this functions appends the ADD to the end with the number

Reply:
works for me. =)